### ISSUE TITLE
No scroll/focus pressing preview on answered message

### ISSUE THREAD

**Issue by Leonscreative** [element-hq/element-x-android#2134]
### Steps to reproduce

When someone or yourself answer a message, when pressing on the answered message preview,itt doesn't change the focus/jump to that message.

### 

### Your phone model

Xiaomi

### Operating system version

MIUI 14.0.9.0 

### Application version and app store

13.tkq

### Homeserver

Matrix.org

### Will you send logs?

No

### Are you willing to provide a PR?

Yes

**Comment by rom4nik** [comment]
Duplicate of #1472?

**Comment by jmartinesp** [comment]
Yes, closing this as a duplicate. It's something we plan to implement in the near future.
